OracleCoherence/README.md

```diff
@@ -1,115 +1,193 @@
-2. Go to the directory containing the Dockerfile for 12.2.1.4
+1. Go to the directory containing the Dockerfile
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker`
+    ```shell script
+    cd OracleCoherence/dockerfiles/14.1.1.0.0/src/main/docker
+    ```
 
-3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the Coherence distribution file of your choice into this folder. The build script supports either building an image from either the Standalone Installer, **fmw_12.2.1.4.0_coherence_Disk1_1of1.zip** or the Quick Installer **fmw_12.2.1.4.0_coherence_quick_Disk1_1of1.zip**
+1. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop
+   the Coherence distribution file of your choice into this folder. The build script supports
+   building an image from either the Quick Installer, `fmw_14.1.1.0.0_coherence_quick_Disk1_1of1.zip`,
+   or the Standalone Installer, `fmw_14.1.1.0.0_coherence_Disk1_1of1.zip`.
 
-    Note that the `Dockerfile` is currently configured to use the Quick Installer.
+    > **_NOTE:_** The `Dockerfile` is currently configured to use the Quick Installer.
 
-4. Go to the maven project directory
+1. Go to the maven project directory
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0`
+    ```shell script
+    cd OracleCoherence/dockerfiles/14.1.1.0.0
+    ```
     
-5. Build the Docker image with Maven
+1. Build the Docker image with Maven
+
+    To build a Docker image using Oracle 8 JDK:
 
-    `$ mvn install`
+    ```shell script
+    mvn install
+    ```
+
+    The resulting image file will be called `oracle/coherence:14.1.1.0.0`.
+
+    To build a Docker image using GraalVM (CE) (*Coherence 14.1.1.0.0 only*)
+
+    ```shell script
+    mvn install -Pgraal
+    ```
+   
+   The resulting image file will be called `oracle/coherence-graal:14.1.1.0.0`.
 
-Follow this procedure for Coherence 12.2.1.3.0 and earlier Coherence versions.
+### To build a Docker image for for Coherence 12.2.1.3.0 and earlier Coherence versions
 
 1. Checkout the GitHub Oracle Docker Images repository
 
-	`$ git clone git@github.com:oracle/docker-images.git`
+    ```shell script
+    git clone git@github.com:oracle/docker-images.git
+    ````
 
-2. Go to the **OracleCoherence/dockerfiles/12.2.1.3** folder
+2. Go to the `OracleCoherence/dockerfiles/12.2.1.3` folder
 
-        $ cd OracleCoherence/dockerfiles/12.2.1.3.0
+    ```shell script
+    cd OracleCoherence/dockerfiles/12.2.1.3.0
+    ```
 
-3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the Coherence distribution file of your choice into this folder. The build script supports either building an image from either the Standalone Installer, **fmw_12.2.1.3.0_coherence_Disk1_1of1.zip** or the Quick Installer **fmw_12.2.1.3.0_coherence_quick_Disk1_1of1.zip**
+3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop
```

Comment:
I would recommend rewriting this line as "`[Download the Coherence distribution file](https://www.oracle.com/middleware/technologies/coherence-downloads.html) of your choice and save it to the current directory`" instead so that the link is more accessible for screen readers.